assets/app/scripts/directives/logViewer.js

```diff
@@ -269,6 +269,11 @@ angular.module('openshiftConsole')
                 streamer = null;
                 $scope.$evalAsync(function() {
                   $scope.autoScrollActive = false;
+                  // if no logs, they have already been archived.
+                  if(lastLineNumber === 0) {
+                    $scope.state = 'empty';
+                    $scope.emptyStateMessage = 'The logs are no longer available or could not be loaded.';
```

Comment:
I'm gonna screenshot & doc a bunch of different cases here to help the discussion along and make sure we land on the right messages.  They will be coming shortly.